MAUI/Autocomplete/Selection.md

```diff
@@ -397,13 +397,13 @@ autocomplete.IsDropDownOpen = true;
 {% endtabs %}
 
 
-### Clear Selected Items
+## Clear Selected Items
 
-You can remove all selected items using the **Clear** method.
+You can use the **Clear** method to remove all selected items in Autocomplete.
```

Comment:
Users can remove selected items and input text using the Clear method in the Autocomplete control.

Reply:
Addressed the comments.